### DISCUSSION TITLE
How to index WMV videos while temporarily skipping exiftool

### DISCUSSION THREAD

**Discussion by kerkhc** [photoprism/photoprism#4207] [Q&A] [answered]
Hi all,
I would like to import a bunch of WMV-files (Windows Media Video).
Doing so, the index process takes a property  "CreationDate": "2017:12:12 19:53:50Z" using exiftool.
Problem: the CreationDate doesn't match the real date (2013-11-15).

a) Is there a way to skip the exiftool for only WMV-files?
(I saw that exiftool can be deactivated in general)

b) How could I use a JSON-file to add e.g. TakenAt, Latitude, Longitude? (I have such information outside in calc-sheet)
If so, is there a specific JSON-structure or is it the same as of Google TakeOut?

Thanks for hints and tips
Carsten

**Comment by graciousgrey** [comment]
It is not possible to skip exiftool only for certain files. 

However you could use exiftool to write date and GPS information into your files. You can find an overview of the fields supported by PhotoPrism here: 

https://www.photoprism.app/kb/metadata
https://docs.photoprism.app/user-guide/library/metadata/

**Reply by kerkhc** [reply]
Hi Theresa,
unfortunately, exiftool only supports read-operation on WMV-files [exiftool Supported File Types](https://exiftool.org/index.html).
Is there a chance to apply a json-file containing my preferred tag values?
Regards,
Carsten

**Reply by graciousgrey** [reply]
You could use the google json format however if the media file itself contains a date in it's metadata this date will be used until the following issue is solved:

- https://github.com/photoprism/photoprism/issues/2133

Alternatively you could use yaml sidecar files in the format of our [yaml export files](https://docs.photoprism.app/user-guide/backups/export/), e.g.

```
TakenAt: 2020-11-08T11:49:01Z
TakenAtLocal: 2020-11-08T12:49:01Z
TakenSrc: manual
TimeZone: Europe/Berlin
Altitude: 35
Lat: 56.523193
Lng: 22.408197
PlaceSrc: manual
Day: 08
Month: 11
Year: 2020
```

Metadata will be read from yaml files during the initial indexing of a file. If you change it later or add a yaml file for a media file that has already been indexed it will not be used.

**Reply by kerkhc** [reply, marked as answer]
I had only partial success by providing .json or .yml files.
As you already mentioned, the metadata inside the WMV-file is still prioritized.

My workaround:
- convert WMV-file to MP4 using ffmpeg (on CLI).
- generate Google format JSON file.
- Upload MP4 and JSON files to photoprism.
Result: all metadata of JSON file is accepted by photoprism

I assume that YML file would work, too. But I have already code to generate JSON...
